sonarts-sq-plugin/sonar-typescript-plugin/src/main/java/org/sonar/plugin/typescript/executable/SonarScannerTSCoreBundleFactory.java

```diff
@@ -23,16 +23,14 @@
 import org.sonar.api.batch.BatchSide;
 import org.sonar.api.batch.ScannerSide;
 import org.sonar.api.config.Configuration;
-import org.sonarsource.api.sonarlint.SonarLintSide;
 
 @BatchSide
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this use of "BatchSide"; it is deprecated. ([squid:CallToDeprecatedMethod](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=sonarts&branch=issue-351&issues=AWD-EPPSEFfjxkQcPti0&open=AWD-EPPSEFfjxkQcPti0)

[ISSUE_KEY]: AWD-EPPSEFfjxkQcPti0

Reply:
This issue has been resolved by a user in SonarQube

[ISSUE_STATUS]: RESOLVED